.gitignore

```diff
@@ -1,6 +1,11 @@
 /sprites/
 /audio/
 /config/
+!/config/config-example.js
+!/config/config-example.inc.php
+!/config/routes.json
+!/config/domain-whitelist.json
+!/config/servers-example.inc.json
```

Comment:
These don't need to be manually added to un-gitignore, by the way. You can just use `git add` to add a gitignored file.

Reply:
Yes, but I think for domain-whitelist specifically if we want it to be versioned we should explicitly allow it to avoid confusion.

Reply:
I also don't really see much reason to gitignore files that are intended to be versioned?